### DISCUSSION TITLE
GAP8 not starting

### DISCUSSION THREAD

**Discussion by Jool99** [bitcraze/discussions#1305] [Q&A] [answered]
Hi! I wonder if anyone has had this same issue with the AI Deck 1.1 and knows how to avoid it.

* Sometimes the GAP8 does not start. The LED next to it does not flash, but the ESP32 LED is flashing.
* It happens mostly after crashing and restarting the drone.
* It sometimes happens after flashing with JTAG and restarting.
* It can be fixed by unplugging battery and replugging it after about 10 sec. Turning the drone off is not enough.
* It is much more likely to happen if the battery is tightly squeezed between the AI Deck and main board. We are using the 350mwh battery with AI Deck on top.

All answers are appreciated! Thanks in advance! :)

**Comment by Jool99** [comment]
Standard STM32 firmware and wifi-image-streamer example on GAP8 flashed with JTAG

**Comment by Jool99** [comment]
Tried with another AI Deck and did not have the same problem.

Is there any hope to try and troubleshoot the broken one? :)

**Reply by evoggy** [reply]
Hi!

Are you running the latest version of the Crazyflie firmware and the ESP32 firmware? This sounds like an issue which has been fixed where we have to hold off between power off/on/off the GAP8.

**Reply by Jool99** [reply]
Hmm not sure. Will check. That is what gets downloaded with the instructions bellow right?

https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#preparing-the-ai-deck-and-crazyflie-firmware

**Reply by evoggy** [reply, marked as answer]
Yes, selecting the latest version of the Crazyflie firmware here will also update the ESP32 if the AI deck is attached.

**Reply by Jool99** [reply]
Thanks for the reply!

This problem disappeared after using:
* Latest firmware from cfclient
* Latest stm32 firmware from repo
* Latest gap8 bootloader repo
* Latest wifi-streaming-example from repo
